src/Symfony/Component/Notifier/Bridge/Slack/SlackOptions.php

```diff
@@ -97,6 +97,10 @@ public function asUser(bool $bool): self
      */
     public function block(SlackBlockInterface $block): self
     {
+        if (50 === \count($this->options['blocks'] ?? [])) {
```

Comment:
Wait they can also be set via the ctor which needs to be checked as well. I did sth similar in the MicrosoftTeamsOptions

Reply:
done